modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -62,12 +62,19 @@ scp your_username@remotehost:/opt/anypoint/runtimefabric/backup.tar.gz /backup-p
 
 == Restore a Cluster
 
+[WARNING]
+====
+Restore must be performed to the same Runtime Fabric Cluster in Control Plane (Same Runtime Fabric cluster in Runtime Manager)
+====
+
 Runtime Fabric provides two target options when restoring a cluster from a backup:
 
-* Use an existing Runtime Fabric cluster.
+* Use an existing Runtime Fabric cluster [underlying infrastructure].
```

Comment:
@aqurishi-mulesoft I'm not sure what this additions means. Can you clarify for me?

Reply:
ok to remove